### PR TITLE
Fix display issues

### PR DESCRIPTION
**Pull request by jameesjohn**
You can comment on the part you don't understand.
I'll try to explain from here.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,32 +1,29 @@
-        let btn = document.querySelector("button");
-        btn.addEventListener("click", countDown);
-               
-        let hours = "0";
-        let minutes = "0";
-        let seconds =  "10";
+let btn = document.querySelector("button");
+btn.addEventListener("click", countDown);
```
Comment (jameesjohn) [inline main.js:2]: Here's a comment for example

File: main.js [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
+   let watch = hours + ':' + minutes + ':' + seconds;
+   countContainer.innerHTML = watch;
+   if(Number(seconds) === 0) {
+      clearInterval(thisInterval);
+   }
+   if(seconds > 10) {
+      seconds -= 1;
+   } else {
+      seconds = Number(seconds) -1;
+      seconds = `0${seconds}`;
```
Comment (AmanamIsrael) [inline main.js:25]: So what does the `0${seconds}` do?

File: main.js [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
-            }
-                    
-            }
-            
\ No newline at end of file
+function countDown() {
+   let thisInterval = setInterval(() => {
+   let watch = hours + ':' + minutes + ':' + seconds;
+   countContainer.innerHTML = watch;
+   if(Number(seconds) === 0) {
```
Comment (AmanamIsrael) [inline main.js:18]: I don't understand this, please

File: main.js (hunk 2 above)
Comment (jameesjohn) [reply main.js:25]: Prints the number with 0 in front

File: main.js (hunk 3 above)
Comment (jameesjohn) [reply main.js:18]: I'm making sure that seconds is a number

**Pull request merged**